### DISCUSSION TITLE
[Projects -  Beta] Add own iteration start date.

### DISCUSSION THREAD

**Discussion by kkroese** [community/community#7464] [Projects and Issues]
It would be great if one could customize the first iteration start date.
It would be useful if you are planning a project to start some time in the future and you want your first sprint (iteration) to start on a specific date.

**Comment by polevaultweb** [comment]
Similar to https://github.com/github/feedback/discussions/6776

**Reply by andreyrd** [reply]
Dates should be fully customizable regardless if it's an existing iteration or a new one. If I create an Iteration late in the week, I still want to mark it as having started on Monday.
